### ISSUE TITLE
Different to UI Harness?

### ISSUE THREAD

**Issue by BevanR** [storybooks/storybook#217]
How is React Storybook different to [UI Harness](https://github.com/philcockfield/ui-harness)?

I read https://github.com/philcockfield/ui-harness/issues/29 already.  A concise summary would be useful for anyone weighing up the two options.

I asked the same question of the UI Harness team; https://github.com/philcockfield/ui-harness/issues/68

**Comment by SimeonC** [comment]
I evaluated both and I think it really boiled down to which way of writing the setup. For me UIHarness reads like a set of TDD scripts whereas Storybook read like scenarios that a non-developer could understand.

I picked Storybook as I liked that I can use the story source code as examples and I didn't like the describe/it abstraction. From reading around their repository it looks like storybook are moving more towards it being docs and demos all in one. Dunno much about UIHarnesses direction.

Plus there's more stars here than there.
